crowbar_framework/app/views/nodes/_node_groups.html.haml

```diff
@@ -7,16 +7,19 @@
     .col-xs-3{ "data-id" => group_name, "data-droppable" => @draggable.to_s, "data-group" => group_name }
       .panel.panel-default
         .panel-heading
-          %h2
-            = piechart_for group
+          %h3
             = truncate(group_name || t("unknown"), :length => 25)
 
         %ul.list-group
           - group[:nodes].values.sort_by { |handle| @nodes[handle][:alias] }.each do |handle|
             - @nodes[handle].tap do |node|
-              %li.list-group-item{ "data-id" => handle.parameterize, "data-draggable" => @draggable.to_s, "data-update" => group_change_path(:id => handle.parameterize, :group => "{0}".html_safe) }
-                %span.led{ :class => node[:status], :title => t(node[:state], :scope => "state", :default => node[:state].titlecase), "data-node" => handle }
-                = link_to(truncate(node[:alias], :length => 25), node_path(handle), :title => node[:description])
+              %li.list-group-item{ "data-id" => handle.parameterize,
```

Comment:
Hash attribute should end with one space before the closing brace<br>Avoid defining `class` in attributes hash for static class names